tests/apps/test_data.py

```diff
@@ -35,6 +35,10 @@
 
 MOLECULE_DATASETS_LIST = [data.sample.Formic]
 
+WATER_DATASETS_LIST = [data.sample.Water]

```

Comment:
Would these all be water-based datasets if this were to be populated more?

Reply:
I don't think it will be populated more. Should we group data differently?

Reply:
I was thinking these could be more like "VIBRATIONAL_DATASETS" but not really a priority at this point.